runtime/compiler/z/codegen/S390J9CallSnippet.cpp

```diff
@@ -527,20 +527,45 @@ TR::S390VirtualUnresolvedSnippet::emitSnippetBody()
    *(uintptrj_t *) cursor = (uintptrj_t) callNode->getSymbolReference()->getCPIndexForVM();
    cursor += TR::Compiler->om.sizeofReferenceAddress();
 
+   // instruction to be patched
    *(uintptrj_t *) cursor = (uintptrj_t) (getPatchVftInstruction()->getBinaryEncoding());
    AOTcgDiag1(comp, "add TR_AbsoluteMethodAddress cursor=%x\n", cursor);
    cg()->addExternalRelocation(new (cg()->trHeapMemory()) TR::ExternalRelocation(cursor, NULL, TR_AbsoluteMethodAddress, cg()),
                           __FILE__, __LINE__, callNode);
    cursor += sizeof(uintptrj_t);
 
+   // Field used by nestmate private calls
+   // J9Method pointer of the callee. Initialized to 0.
+   *(uintptrj_t *) cursor = 0;
+   cursor += sizeof(uintptrj_t);
+
+   // Field used by nestmate private calls
+   // J2I thunk address
+   // TODO: leave out the AOT until AOT supports this.
```

Comment:
Similarly to the comment made in the OMR PR do we have an issue opened for this that we can reference?

Reply:
Updated this pull request with a new commit that removes these constructors because they are unused and takes no thunkAddress parameter.